### DISCUSSION TITLE
Lithuania language is missing

### DISCUSSION THREAD

**Discussion by Armikas** [pndurette/gTTS#475] [Q&A]
Hello, I`m from Lithuania.
Lithuanian language is missing
In https://github.com/pndurette/gTTS/blob/main/gtts/langs.py there is no Lithuanian language.
It would be nice to add Lithuanian language along many others.
Can you fix this issue?

**Comment by pndurette** [comment]
This was a bug in the language updater. It was fixed and Lithuanian is available in v2.5.2!

**Reply by Armikas** [reply]
Hi, pndurette,
I can`t get it working in Home Assistant...
I`m confused. Could you take a look, please: https://github.com/home-assistant/core/issues/111909
